src/components/Notification/Notification.stories.mdx

```diff
@@ -43,40 +77,189 @@ Notifications are used to display global information, our notification variants
+  >
+    {Template.bind({})}
   </Story>
 </Canvas>
 
-### Information
+### Inline
+
+When vertical space is limited, you can use the inline variant.
 
 <Canvas>
-  <Story name="Information">
-    <Notification type="information" status="Information:">
-      Anyone with access can view your invited users.
-    </Notification>
+  <Story
+    name="Inline"
+    args={{
+      children: "Not enough space.",
+      inline: true,
+      severity: "negative",
+      title: "Error:",
+    }}
+  >
+    {Template.bind({})}
   </Story>
 </Canvas>
+
+### Borderless
+
+In cases where a notification sits inside another component, such as a table cell or a card, it may be useful to remove the outer border and highlight bar.
+
+<Canvas>
+  <Story
+    name="Borderless"
+    args={{
+      borderless: true,
+      children: "Only 8GB storage remaining.",
+      severity: "caution",
+      title: "Warning",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Actions
+
+Actions will appear below the notification message.
+
+<Canvas>
+  <Story
+    name="Actions"
+    args={{
+      actions: [
+        { label: "Action 1", onClick: () => null },
+        { label: "Action 2", onClick: () => null },
+      ],
+      children:
+        "Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.",
+      severity: "information",
+      title: "Title",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Dismissible
+
+Notifications that can be dismissed should be given an onDismiss function.
+
+<Canvas>
+  <Story name="Dismissible">
+    {() => {
+      const [show, setShow] = useState(true);
+      if (!show) {
+        return null;
+      }
+      return (
+        <Notification onDismiss={() => setShow(false)} title="Dismissible">
+          Click the dismiss button to get rid of me.
+        </Notification>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Timeout
+
+Notifications can be automatically dismissed after a timeout interval.
+
+<Canvas>
+  <Story name="Timeout">
+    {() => {
+      const [show, setShow] = useState(true);
+      const [timer, setTimer] = useState(0);
+      if (!show) {
+        return null;
+      }
+      return (
+        <Notification
+          actions={[{ label: "Start timer", onClick: () => setTimer(3000) }]}
+          onDismiss={() => setShow(false)}
+          timeout={timer}
+          title="Timeout"
+        >
+          Click the action to start a 3 second timer.
+        </Notification>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Timestamp
+
+For notifications in which recency is important, you can include a section for time.
+
+<Canvas>
+  <Story
+    name="Timestamp"
+    args={{
+      children:
+        "Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.",
+      severity: "information",
+      timestamp: "1h ago",
+      title: "Title",
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Deprecated
```

Comment:
I'd suggest moving this up, just under the generated props list.

Reply:
:+1: